### PR TITLE
[JA] do not add whitespaces between Latin and Japanese

### PR DESCRIPTION
**Pull request by marocchino**
Command was:
```
perl -i -p -e 's/(\w+) ([一-龯ぁ-んァ-ン。、])/$1$2/g;' *.md */*/*.md
perl -i -p -e 's/([一-龯ぁ-んァ-ン。、]) (\w+)/$1$2/g;' *.md */*/*.md
```

### CODE REVIEW

File: ja/lessons/basics/documentation.md [hunk 1]
```diff
@@ -102,11 +102,11 @@ iex> h Greeter.hello
 
                 def hello(name)
 
-Hello メッセージを表示します
+Helloメッセージを表示します
 
 パラメータ
 
-  • name: 人名を表現する文字です。
+  •name: 人名を表現する文字です。
```
Comment (marocchino) [inline ja/lessons/basics/documentation.md]:
```suggestion
  • name: 人名を表現する文字です。
```

File: ja/lessons/basics/documentation.md [hunk 2]
```diff
@@ -154,18 +154,18 @@ $ cd greet_everyone
...
```
Comment (marocchino) [inline ja/lessons/basics/documentation.md]:
```suggestion
  • name: 人名を表現する文字です。
```

File: ja/lessons/basics/iex-helpers.md [hunk 3]
```diff
@@ -84,14 +84,14 @@ iex> h Enum
 Provides a set of algorithms that enumerate over enumerables according to the
 Enumerable protocol.
 
-┃ iex> Enum.map([1, 2, 3], fn(x) -> x * 2 end)
+┃iex> Enum.map([1, 2, 3], fn(x) -> x * 2 end)
```
Comment (marocchino) [inline ja/lessons/basics/iex-helpers.md]:
```suggestion
┃ iex> Enum.map([1, 2, 3], fn(x) -> x * 2 end)
```

File: ja/lessons/basics/iex-helpers.md [hunk 4]
```diff
@@ -84,14 +84,14 @@ iex> h Enum
...
 ┃ [2, 4, 6]
 
 Some particular types, like maps, yield a specific format on enumeration. For
 example, the argument is always a {key, value} tuple for maps:
 
-┃ iex> map = %{a: 1, b: 2}
-┃ iex> Enum.map(map, fn {k, v} -> {k, v * 2} end)
+┃iex> map = %{a: 1, b: 2}
```
Comment (marocchino) [inline ja/lessons/basics/iex-helpers.md]:
```suggestion
┃ iex> map = %{a: 1, b: 2}
```

File: ja/lessons/basics/iex-helpers.md [hunk 5]
```diff
@@ -84,14 +84,14 @@ iex> h Enum
...
+┃iex> Enum.map(map, fn {k, v} -> {k, v * 2} end)
```
Comment (marocchino) [inline ja/lessons/basics/iex-helpers.md]:
```suggestion
┃ iex> Enum.map(map, fn {k, v} -> {k, v * 2} end)
```

File: ja/lessons/basics/iex-helpers.md [hunk 6]
```diff
@@ -102,11 +102,11 @@ Since the majority of the functions in Enum enumerate the whole enumerable and
 return a list as result, infinite streams need to be carefully used with such
 functions, as they can potentially run forever. For example:
 
-┃ Enum.each Stream.cycle([1, 2, 3]), &IO.puts(&1)
+┃Enum.each Stream.cycle([1, 2, 3]), &IO.puts(&1)
```
Comment (marocchino) [inline ja/lessons/basics/iex-helpers.md]:
```suggestion
┃ Enum.each Stream.cycle([1, 2, 3]), &IO.puts(&1)
```

File: ja/lessons/basics/iex-helpers.md [hunk 7]
```diff
@@ -144,7 +144,7 @@ Kernel.struct/2.
 
 Examples
 
-┃ iex> Map.merge(%{a: 1, b: 2}, %{a: 3, d: 4})
+┃iex> Map.merge(%{a: 1, b: 2}, %{a: 3, d: 4})
```
Comment (marocchino) [inline ja/lessons/basics/iex-helpers.md]:
```suggestion
┃ iex> Map.merge(%{a: 1, b: 2}, %{a: 3, d: 4})
```

File: ja/lessons/basics/modules.md [hunk 8]
```diff
@@ -58,12 +58,12 @@ defmodule Example do
...
 
-重要なので言及しておきますと、 Elixir には予約されている属性があります。
-もっとも一般的なのは以下の 3 つです:
+重要なので言及しておきますと、Elixirには予約されている属性があります。
+もっとも一般的なのは以下の3つです:
 
 - `moduledoc` — 現在のモジュールにドキュメントを付けます。
 - `doc` — 関数やマクロについてのドキュメント管理。
-- `behaviour` — OTP またはユーザが定義した振る舞い(ビヘイビア)に用います。
+- `behaviour` —OTPまたはユーザが定義した振る舞い(ビヘイビア)に用います。
```
Comment (marocchino) [inline ja/lessons/basics/modules.md]:
```suggestion
- `behaviour` — OTPまたはユーザが定義した振る舞い(ビヘイビア)に用います。
```

File: ja/lessons/advanced/erlang.md [hunk 9]
```diff
@@ -23,23 +23,23 @@ defmodule Example do
 end
 
 iex> Example.timed(fn (n) -> (n * n) * n end, [100])
-Time: 8 μs
+Time: 8μs
```
Comment (marocchino) [inline ja/lessons/advanced/erlang.md]:
```suggestion
Time: 8 μs
```

File: ja/lessons/basics/documentation.md (hunk 1 above)
Comment (starfish719) [reply ja/lessons/basics/documentation.md]: @marocchino It is better to put whitespace 👍

**Pull request merged**